tests/xcodeproj/test.swift

```diff
@@ -4,5 +4,12 @@ class SwiftTests : XCTestCase {
   func testPasses() {
     XCTAssertTrue(true)
   }
+  func testTestEnvArgsMatches() {
+    XCTAssertEqual(ProcessInfo.processInfo.environment["test_envvar_key1"], "test_envvar_value1")
```

Comment:
This is fine to add here, but FYI, we aren't actually running these tests in CI.

I will probably run these tests as part of CI in a separate PR.

Reply:
I am actually considering to try to add this into CI in this PR (otherwise my change has no effect)